src/Classes/ServiceAPI/MyRadio_Podcast.php

```diff
@@ -174,26 +174,32 @@ protected function __construct($podcast_id)
     /**
      * Get all the Podcasts that the User is Owner of Creditor of.
      *
-     * @param MyRadio_User $user Default current user.
+     * @param int|string|MyRadio_User $user Default current user.
      *
      * @return MyRadio_Podcast[]
      */
-    public static function getPodcastsAttachedToUser(MyRadio_User $user = null)
+    public static function getPodcastsAttachedToUser($user = null)
     {
         return self::resultSetToObjArray(self::getPodcastIDsAttachedToUser($user));
     }
 
     /**
      * Get the IDs of all the Podcasts that the User is Owner of Creditor of.
      *
-     * @param MyRadio_User $user Default current user.
+     * @param int|string|MyRadio_User $user Default current user.
      *
      * @return int[]
      */
-    public static function getPodcastIDsAttachedToUser(MyRadio_User $user = null)
+    public static function getPodcastIDsAttachedToUser($user = null)
     {
         if ($user === null) {
             $user = MyRadio_User::getInstance();
+        } elseif (is_int($user)) {
```

Comment:
May just be GitHub mobile, but is the indentation correct here?

Reply:
Looks fine to me